rules/windows/builtin/win_remote_powershell_session.yml

```diff
@@ -1,6 +1,6 @@
-title: Remote PowerShell Sessions
+title: Remote PowerShell Sessions Network Connections (WinRM)
 id: 13acf386-b8c6-4fe0-9a6e-c4756b974698
-description: Detects basic PowerShell Remoting by monitoring for network inbound connections to ports 5985 OR 5986
+description: Detects basic PowerShell Remoting (WinRM) by monitoring for network inbound connections to ports 5985 OR 5986
 status: experimental
 date: 2019/09/12
```

Comment:
i think you can add the modified line with the date of your change 

like this 

```
modified: 2021/05/21
```